pydash/pydash_web/controller/register_user.py

```diff
@@ -58,6 +60,20 @@ def register_user():
                                  email_address,
                                  user.name)
 
+
+_DEFAULT_EXPIRATION_TIME = timedelta(days=1)
+
+
+class VerificationCode:
```

Comment:
this class does not belong here

Reply:
Agreed, but this was already here when I changed stuff. I will not change this but change it in a different PR, as this one is only about the mail stuff.

Reply:
This class is also different from the `VerificationCode` class in `pydash_app`. What use does it have? Furthermore, having two classes that have the same name, are nearly identical, but differ in a key aspect, seems like a very dangerous thing indeed.

Reply:
@TheVeggydude that's weird, it shows up as added in this PR.

I agree with @TWEApol, this smells like code duplication